.github/workflows/commit_manifest.yml

```diff
@@ -3,6 +3,10 @@ on:
   push:
     branches:
       - "main"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
```

Comment:
I don't know if `github.ref` needed here. I think just `github.workflow` is the correct group. If `ref` is always `main` it shouldn't matter but I'm not 100% thats the case as it might refer to the branch being merged in? I would double check to be certain.